lib/blocklist.js

```diff
@@ -0,0 +1,48 @@
+/* Any copyright is dedicated to the Public Domain.
```

Comment:
General policy is MPL by default, unless we have a specific reason to use a different license.